lib/matplotlib/axis.py

```diff
@@ -1646,6 +1652,11 @@ def set_minor_formatter(self, formatter):
         formatter : `~matplotlib.ticker.Formatter`
         """
         cbook._check_isinstance(mticker.Formatter, formatter=formatter)
+        if (isinstance(formatter, mticker.FixedFormatter)
+                and len(formatter.seq) > 0
+                and not isinstance(self.minor.locator, mticker.FixedLocator)):
+            warnings.warn('FixedFormatter should only be used together with '
```

Comment:
This one too?

Reply:
:+1: One shouldn't try to fix things on the go...